### ISSUE TITLE
Prediction and forecast for Multiple input multipleoutput system

### ISSUE THREAD

**Issue by prajwalthakur** [jxx123/fireTS#10]
I know that there is a way for training a MIMO system by training multiple MISO systems and get multiple models. But is there any way to get only a single model?

**Comment by jxx123** [comment]
As far as I know, you can construct a single neural network with multiple outputs to do that. fireTS does not support a single model with multiple outputs. 

I think the major difference between multiple MISO models and single MIMO model is that the single MIMO model shares part of the parameters for the multiple outputs, which is not considered when I wrote fireTS package.

**Comment by prajwalthakur** [comment]
yes thats true "single MIMO model shares part of the parameters for the multiple outputs". I guess fireTS does not support the generation of MIMO  training and test data set? Right now I guess it only support generation training and test data for MISO system.

**Comment by jxx123** [comment]
Yes, the package only supports MISO case.
